interface/modules/custom_modules/oe-module-faxsms/src/Controller/AppDispatch.php

```diff
@@ -49,6 +50,7 @@ public function __construct($type = null)
         if (empty(self::$_apiModule)) {
             self::$_apiModule = $_REQUEST['type'] ?? $_SESSION["oefax_current_module_type"] ?? null;
         }
+        $this->crypto = new CryptoGen();
```

Comment:
Was there a reason instantiating this was deferred to the sub classes?  It seemed odd not to just do it here, since all sub classes would need it for `saveSetup()`.